### PR TITLE
Remove mstriemer and update staging push documentation

### PR DESCRIPTION
**Pull request by muffinresearch**
Fix push docs and remove @mstriemer from amo push duty.

### CODE REVIEW

File: docs/server/push-duty.rst [hunk 1]
```diff
@@ -134,32 +133,17 @@ Then update the etherpad with the new comparison link for the updated tag.
...
-Consult the IRC bot docs for more up to date information but here are examples
-of commands you would run to deploy all the apps to stage:
+Our infrastructure automates pushing the tags to stage once the tags have
+been pushed up to the repository.
 
-* ``!build addons-stage YYYY.MM.DD``
-* ``!build admin-stage YYYY.MM.DD``
-* ``!build discopane-stage YYYY.MM.DD``
-* ``!build versioncheck-stage YYYY.MM.DD``
+The only required step is to check the tag has deployed by looking out for
```
Comment (tofumatt) [inline docs/server/push-duty.rst]: 🎉 🎉 🎉

**Comment by muffinresearch** [comment]
Hmm I might have to add some skips to the datadog links...

File: docs/server/push-duty.rst [hunk 2]
```diff
@@ -134,32 +133,17 @@ Then update the etherpad with the new comparison link for the updated tag.
...
-message. Do not be fooled! This doesn't mean the change is live. You need to
-wait and watch for each ``amo-xxx-Promote succeeded`` message in ``#remora``.
-The *Promote* message indicates that it's live on the server.
-Here is an example of a *Promote* message to indicate the changes are live
-on the server::
-
-    <cloudops-hubot> amo-olympia-web-STAGE-Promote build #156 succeeded (https://deploy.mozaws.net/job/amo-olympia-web-STAGE-Promote/156/)
-
-Jason can help with this if needed.
+ * `Addons Server (stage) <https://addons.allizom.org/__version__>`_
```
Comment (tofumatt) [inline docs/server/push-duty.rst]: Thanks for including the links! (they're annoying to type and remember)

File: docs/server/push-duty.rst [hunk 3]
```diff
@@ -134,32 +133,17 @@ Then update the etherpad with the new comparison link for the updated tag.
...
-* ``!build addons-stage YYYY.MM.DD``
-* ``!build admin-stage YYYY.MM.DD``
-* ``!build discopane-stage YYYY.MM.DD``
-* ``!build versioncheck-stage YYYY.MM.DD``
+The only required step is to check the tag has deployed by looking out for
+the automated push messages in the irc channel.
 
-Each of these apps live in the code respositories that are listed for tagging up
-above.
+You can also check ``/__version`` on a given service which shows the currently
```
Comment (mstriemer) [inline docs/server/push-duty.rst]: __version__

Comment (muffinresearch) [inline docs/server/push-duty.rst]: Ah thanks, good catch.

**Pull request merged**